src/pages/Profile.tsx

```diff
@@ -124,6 +126,68 @@ const Profile = () => {
   });
   const [editProfileModalOpen, setEditProfileModalOpen] = useState(false);
   const [loading, setLoading] = useState(true);
+  const [followListModalLoading, setFollowListModalLoading] = useState(true);
+  const [followListModalOpen, setFollowListModalOpen] = useState(false);
+  const [followListModalUserData, setFollowListModalUserData] = useState<
+    NetworkUsers[]
+  >([]);
+  const [isFollowersModal, setIsFollowersModal] = useState(false);
+  const [isFollowingModal, setIsFollowingModal] = useState(false);
```

Comment:
I don't think we need both of these along with `followListModalOpen`. We can make a `followerListModalOpen` and `followingListModalOpen` to cut it down to 2 states

Reply:
Sure thing.